app/controllers/steps_controller.rb

```diff
@@ -333,92 +331,72 @@ def checklistitem_state
 
   # Complete/uncomplete step
   def toggle_step_state
```

Comment:
this function is not working properly it returns an 404 error with ` {style: "danger"} ` response body

Reply:
have you fixed this?